src/components/resources/Resources.js

```diff
@@ -26,6 +30,14 @@ const Resources = () => {
 
     const [resources, setResources] = React.useState([]);
 
+    // Filters
+    const [mediaTag, setMediaTag] = React.useState('');
+    const mediaTagList = ["Interview Tips", "Youtube", "Stack Overflow", "School Website", "Personal Website", "Spreadsheet"];
```

Comment:
What are your thoughts on changing "School Website" to "School". I feel like that's a bit more inclusive and general